### PR TITLE
Detect 9 Script Changes

### PR DESCRIPTION
**Pull request by devmehtasynopsys**
These changes are related to generation of detect 9 scripts. After referring this document [Detect Scripts](https://sig-confluence.internal.synopsys.com/display/integration/Detect+scripts) by Steve Billings, I have added the previous detect8 scripts to the earlierversions folder and updated DETECT_MAJOR_VERSION to 9.

### CODE REVIEW

**Review by dterrysynopsys — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -8,10 +8,10 @@ Synopsys Detect scripts to invoke Detect.
 ## Where can I get the latest release? ##
 
 *Available from GitHub for Linux by running:*  
-bash <(curl -s -L https://detect.synopsys.com/detect7.sh)
+bash <(curl -s -L https://detect.synopsys.com/detect9.sh)
 
 *Available from GitHub for Windows by running:*  
-powershell "[Net.ServicePointManager]::SecurityProtocol = 'tls12'; irm https://detect.synopsys.com/detect7.ps1?$(Get-Random) | iex; detect"
+powershell "[Net.ServicePointManager]::SecurityProtocol = 'tls12'; irm https://detect.synopsys.com/detect9.ps1?$(Get-Random) | iex; detect"
```
Comment (gopannabd) [inline README.md:14]: Interesting that this change is going from 7 to 9. Was it not updated when 8 was released?

File: src/main/resources/templates/detect-sh.sh [hunk 2]
```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
 
-echo "Detect Shell Script ${SCRIPT_VERSION}"
-
```
Comment (gopannabd) [inline src/main/resources/templates/detect-sh.sh:4]: Was this echo pre-existing? If yes, I would leave it.

Comment (devmehtasynopsys) [reply src/main/resources/templates/detect-sh.sh:4]: Yes it was pre-existing, but the problem is that it is getting echoed 2 times which is not looking good. As per discussion with Mayank removing one of them. The second one is on line 87 'echo "Detect Shell Script ${SCRIPT_VERSION}"'

File: README.md (hunk 1 above)
Comment (devmehtasynopsys) [reply README.md:14]: Yes someone might have missed it in Detect 8 release

**Review by gopannabd — APPROVED** [review]

**Pull request merged**